ick/runner.py

```diff
@@ -70,16 +70,21 @@ def __init__(self, rtc: RuntimeConfig, repo: Repo) -> None:
 
     def iter_rule_impl(self) -> Iterable[BaseRule]:
         name_filter = re.compile(self.rtc.filter_config.name_filter_re).fullmatch
+        rules_matched = 0
         for rule in self.rules:
             if rule.urgency < self.rtc.filter_config.min_urgency:
                 continue
 
             if not name_filter(rule.qualname):
                 continue
 
+            rules_matched += 1
             i = get_impl(rule)(rule)
             yield i
 
+        if rules_matched == 0 and len(self.rules) > 0:
+            print(f"[red]No rules found with urgency `{self.rtc.filter_config.min_urgency.value}` or greater that matches the pattern`{self.rtc.filter_config.name_filter_re}`.[/red]")
+
```

Comment:
The backticks here are a bit odd (and need a space after "pattern").  I think plain quotes will be better.